lte/gateway/c/core/oai/test/ngap/test_ngap_state_converter.cpp

```diff
@@ -0,0 +1,268 @@
+/**
+ * Copyright 2021 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "lte/gateway/c/core/oai/test/ngap/mock_utils.h"
+#include "lte/gateway/c/core/oai/test/ngap/util_ngap_pkt.h"
+#include <gtest/gtest.h>
+
+extern "C" {
+#include "lte/gateway/c/core/oai/common/log.h"
+#include "Ngap_NGAP-PDU.h"
+#include "lte/gateway/c/core/oai/tasks/ngap/ngap_amf_handlers.h"
+#include "lte/gateway/c/core/oai/include/amf_config.h"
+}
+
+#include "lte/gateway/c/core/oai/tasks/ngap/ngap_state_converter.h"
+#include "lte/gateway/c/core/oai/tasks/ngap/ngap_state_manager.h"
+#include "lte/gateway/c/core/oai/include/map.h"
+
+using ::testing::Test;
+
+namespace magma5g {
+
+class NgapStateConverterTest : public testing::Test {
+ protected:
+  void SetUp() {
+    itti_init(
+        TASK_MAX, THREAD_MAX, MESSAGES_ID_MAX, tasks_info, messages_info, NULL,
+        NULL);
+
+    amf_config_init(&amf_config);
+    amf_config.plmn_support_list.plmn_support_count          = 1;
+    amf_config.plmn_support_list.plmn_support[0].s_nssai.sst = 0x1;
+
+    ngap_state_init(2, 2, true);
```

Comment:
should be ngap_amf_init and pass amf_config object

Reply:
Addressed review comment